tests/Unit/CMakeLists.txt

```diff
@@ -3,9 +3,10 @@
 
 set(executable RunTests)
 
-set(SPECTRE_TESTS "Test_TestingFramework.cpp")
-
-set(SPECTRE_COMPILATION_TESTS "TestCompilationFramework.cpp")
```

Comment:
keep this here in case subdirectories add more of them (currently doesn't happen, but may in the future)